packages/@aws-cdk/aws-redshift-alpha/lib/cluster.ts

```diff
@@ -454,6 +461,16 @@ export class Cluster extends ClusterBase {
    */
   public readonly secret?: secretsmanager.ISecret;
 
+  /**
+   *  A default AWS Identity and Access Management (IAM) role to be used by the cluster to access other AWS services.
+   */
+  public readonly serviceRole: iam.IRole;
```

Comment:
```suggestion
  /**
   * A default AWS Identity and Access Management (IAM) role to be used by the cluster to access other AWS services.
   */
  private readonly serviceRole: iam.IRole;
```
No need to make this public.

Reply:
I think it's useful to either make this public or add additional methods to allow users to add permissions to the role directly. 

It's convenient for adding additional permissions beyond what grants provide. For example if I wanted to allow my cluster to read from a Glue Data Catalog.